lib/ash/resource/change/builtins.ex

```diff
@@ -50,16 +68,30 @@ defmodule Ash.Resource.Change.Builtins do
 
     Use `arg(:argument_name)` to use the value of the given argument. If the argument is not supplied then nothing happens.
 
+  ## Options
+
+  #{Spark.OptionsHelpers.docs(@set_attribute_opts)}
```

Comment:
```suggestion
  #{Spark.OptionsHelpers.docs(Keyword.drop(@set_attribute_opts, [:attribute, :value]))}
```

Since they are passed as arguments we can hide them from docs.